gravityflow/gravityflow
# Integrate with TranslationsPress to provide translations

## Description
This PR integrates [TranslationsPress](https://translationspress.com/) with Gravity Flow and all extensions so that all translations can be hosted with TranslationsPress.

This resolves [gravityflow/backlog#89](https://github.com/gravityflow/backlog/issues/89).

## Testing instructions
1. Switch to a language other than English.
2. Install a new Gravity Flow Extension
3. Verify that the extension is in the new language.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
